sdk/src/record-provider.ts

```diff
@@ -255,11 +314,11 @@ class NetworkRecordProvider implements RecordProvider {
      * const programManager = new ProgramManager("https://api.explorer.provable.com/v1", keyProvider, recordProvider);
      * programManager.transfer(1, "aleo166q6ww6688cug7qxwe7nhctjpymydwzy2h7rscfmatqmfwnjvggqcad0at", "public", 0.5);
      */
-    async findCreditsRecord(microcredits: number, unspent: boolean, nonces?: string[], searchParameters?: RecordSearchParams): Promise<RecordPlaintext> {
+    async findCreditsRecord(microcredits: number, searchParameters: RecordSearchParams,nonces?: string[]): Promise<OwnedRecord> {
         let records = null;
 
         try {
-            records = await this.findCreditsRecords([microcredits], unspent, nonces, searchParameters);
+            records = await this.findCreditsRecords([microcredits], searchParameters, nonces);
```

Comment:
Let's put nonces inside the search parameters.

```suggestion
            records = await this.findCreditsRecords([microcredits], searchParameters);
```

Reply:
done